ipn/localapi/localapi.go

```diff
@@ -163,7 +163,24 @@ func Register(name string, fn LocalAPIHandler) {
 	if _, ok := handler[name]; ok {
 		panic("duplicate LocalAPI handler registration: " + name)
 	}
-	handler[name] = fn
+	handler[name] = loggerHandler(name, fn)
+}
+
+// loggerHandler wraps a LocalAPIHandler to log mutating requests.
+func loggerHandler(name string, fn LocalAPIHandler) LocalAPIHandler {
```

Comment:
I'd rather not do this wrapper thing and instead just log at the place where the handlers are looked up and called.